PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,42 @@
-<!-- Your title above should be a short description of what
-was changed. Do not include the issue number in the title. -->
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- Your title above should be a short description of what was changed. Do not
+  include the issue number in the title.
+
+- Development information is available at
+  https://github.com/sympy/sympy/wiki/Development-workflow.
+
+- Do not create the PR out of master, but out of a separate branch.
+
+- Try to keep the number of commits to a minimum. Use rebase and
+  force pushing in git to avoid adding unnecessary commits.
+
+- If this is your first commit, see:
+  https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file
+
+- If you are contributing fixes to docstrings, please pay attention to
+  https://docs.sympy.org/dev/guides/contributing/documentation-style-guide.html.
+  In particular, note the difference between using single backquotes,
+  double backquotes, and asterisks in the markup.
+
+- Please include tests of any new feature or bug fix. The code in the tests
+  should not execute correctly in a version of SymPy without your code.
+
+Note that all reviewing is done an a volunteer basis, so it can sometimes take
+time before you get feedback. If you have not heard back in two weeks, feel
+free to ping @sympy/developers-with-push-access-to-everything
+Do not ping individual persons to get a review, nor request reviews.
+Also note that any reviewer will be notified of changes in your PR, so there
+is no need to ping after a change.
```

Comment:
This is not true. GitHub does not send emails for pushes, only comments. So actually the opposite of this is the case. If you push new changes and want another review, you will need to add a comment to that effect.

Reply:
Actually I just realized that GitHub *does* send emails for pushes. I have them disabled, though, as it's too much noise. Personally I think it's generally good to make an explicit update comment after making a significant change.

Reply:
And it is possible to disable mentions as well. But that is not something that I really would like to do.

> after making a significant change.

That is the thing. Significant change. Even "did half of what you asked for" would in many cases be a step forward.

So far, based on the discussion, the only way out here for me personally is to be less active. While not as active as e.g. @oscarbenjamin and @smichr I get enough pinging to find it a problem enough to open this issue. 

If you do not show up, you will not be pinged.